web/services/github_issue_manager/test_search_for_matching_stacktrace.py

```diff
@@ -0,0 +1,66 @@
+from django.test import TestCase
+from services.models import ErrorReport, GithubIssue
+from services.github_issue_manager.github_issue_manager import _search_for_matching_stacktrace
+
+
+class MatchingStackTraceSearchTest(TestCase):
+    entries = [
```

Comment:
I think the following would not be matched by the regex:

`  File "/opt/mantidworkbench6.9/lib/python3.10/site-packages/mantid/simpleapi.py", line 1083, in __call__`

and also the following kind of thing that I found in the error reports slack channel:

```
Exception: MDWorkspace::getExperimentInfo(): expInfoIndex is out of range.
  at line 152 in '/usr/local/anaconda/envs/mantid-dev/plugins/python/algorithms/ConvertWANDSCDtoQ.py'
```